src/i18n/cs_CZ/buildings.ts

```diff
@@ -585,19 +663,19 @@ export default {
     21: {
         caption: 'Pyrotechnická služba PČR',
         color: '#663300',
-        coins: 50,
-        credits: 5_000_000,
+        coins: 35,
+        credits: 200_000,
         levelPrices: {
-            credits: [10_000, 50_000, ...Array(18).fill(100_000)],
-            coins: [10, 15, ...Array(18).fill(20)],
+            credits: [10_000, 50_000, ...Array(3).fill(100_000)],
+            coins: [10, 15, ...Array(3).fill(20)],
         },
         extensions: [],
-        levelcost: ['1. 10.000', '2. 50.000', '3.-20. 100.000'],
-        maxBuildings: 'No limit',
+        levelcost: ['1. 10.000', '2. 50.000', '3.-5. 100.000'],
+        maxBuildings: 'Bez limitu',
         maxLevel: 5,
         special: '',
         startPersonnel: 4,
-        startVehicles: ['Vozidlo pyrotechnika PČR'],
+        startVehicles: [],
```

Comment:
Building without start vehicle? That is, at least in the german version, unusual

Reply:
yep no start vehicle ^^